kubernetes/resource_kubernetes_persistent_volume_claim_test.go

```diff
@@ -781,57 +793,6 @@ func testAccKubernetesPersistentVolumeClaimConfig_metaModified(name string) stri
 `, name)
 }
 
-func testAccKubernetesPersistentVolumeClaimConfig_import(volumeName, claimName, diskName, zone string) string {
```

Comment:
This change is related to a PR that merged recently, where the imports for all of the acceptance tests were moved into the _basic test for that resource. `tfproviderlint` complains if we have separate tests for imports.